configs/setups/awiesm/awiesm.yaml

```diff
@@ -338,28 +379,36 @@ fesom:
             "*":
                 is: invalid
 
+        # use_ciso switch
+        choose_recom.use_ciso:
+                true:
+                        add_config_sources: 
+                                io: "${namelist_dir}/namelist.io.recom.ciso"
+                                oce: "${namelist_dir}/namelist.oce.recom.ciso"
```

Comment:
We can delete this block by creating a `ciso_namelist_sufix` and setting it to `.ciso` for ciso simulations and `""` for simulations without ciso.